### DISCUSSION TITLE
New instance only displays logo

### DISCUSSION THREAD

**Discussion by viralpoet** [mastodon/mastodon#20460] [General]
I'm setting up a new instance and currently I have the wepage showing the mastodon logo and nothing else. I can't login or if I go to admin/dashboard it says I don't have permission.

If I CAT my error log, I'm getting heaps of permission denied errors from /home/mastodon/live/public/
the folder is drwxrwxr-x 

I've followed the instructions here https://docs.joinmastodon.org/admin/install/ with the only difference being I've gone for Ruby 3.0.4
and updated the config file to reflect that.

**Comment by MrGeneration** [comment]
`/home/mastodon/live/public/` is owned by `mastodon` ?
`775` looks quite open for your directory.

**Comment by viralpoet** [comment]
yes the entire directory from home is owned by mastodon.
I've spun up another instance and followed slightly different instructions but with a similar outcome. Nothing is templated but I can at least see a login.

**Comment by masterdanzh** [comment]
Same issue here.
Everything runs fine but the webpage simply shows the logo, nothing else...

**Reply by svvvst** [reply]
What is the last step you took? Your main mastodonserver.com/about is where this page is?

Did you start mastodon-web? 

I just set up an instance this week and vaguely recall that I might have had an issue like this. Maybe check that nginx is set up correctly.

**Reply by masterdanzh** [reply]
SOLVED:
It was a permission issue.

**Reply by viralpoet** [reply]
All the advice on the net is to make sure home or public directory is publicly visible. And of course they are. But the issue comes about because certain repositories protect the /home path from being accessible.
I also found a permission issue, but to locate it I went from home to home/mastodon/live/public  backwards until I found 1 directory with an incorrect permission. From memory it was `/home/mastodon/`

**Reply by masterdanzh** [reply]
I solved it by including www-data into the mastodon group and viceversa.
